src/ngraph/runtime/cpu/mkldnn_emitter.cpp

```diff
@@ -2099,3 +2272,831 @@ size_t MKLDNNEmitter::build_quantized_inner_product_forward(
     m_primitive_deps[ip_index] = {input_data_index, weights_index, result_index};
     return ip_index;
 }
+
+// Translate inputs of \p node to TensorViewWrappers.
+static void translate_node_inputs(const Node* node, std::vector<TensorViewWrapper>& args)
```

Comment:
With these `translate_*`, I'm trying to keep the original code as it was but I'm not sure if creating TensorViewWrapper is actually needed at this point. Could someone provide feedback on this?